### PR TITLE
Issue templates

### PR DESCRIPTION
**Pull request by jmklix**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 1]
```diff
@@ -5,13 +5,6 @@ title: "(short issue description)"
 labels: [feature-request, needs-triage]
 assignees: []
 body:
-  - type: markdown 
-    attributes: 
-      value: |
-        ## Community Note
-        1. Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
-        2. Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
-        3. If you are interested in working on this issue, please leave a comment
```
Comment (waahm7) [inline .github/ISSUE_TEMPLATE/feature-request.yml:14]: Trivial: crt-swift is in dev preview, we might want to keep this.

**Review by waahm7 — APPROVED** [review]

File: .github/workflows/stale_issue.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+name: "Close stale issues"
```
Comment (waahm7) [inline .github/workflows/stale_issue.yml:1]: We can’t auto close stale issues as crt-swift uses issues for planning/tracking purposes.

**Review by waahm7 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/feature-request.yml (hunk 1 above)
Comment (jmklix) [reply .github/ISSUE_TEMPLATE/feature-request.yml]: readded

File: .github/workflows/stale_issue.yml (hunk 2 above)
Comment (jmklix) [reply .github/workflows/stale_issue.yml:1]: commented out the part that will close issues at one year. Left in the bot that closes issues after response requested

File: .github/workflows/stale_issue.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        response-requested-label: response-requested
+
+        # Don't set closed-for-staleness label to skip closing very old issues
+        # regardless of label
+        # closed-for-staleness-label: closed-for-staleness
+
+        # Issue timing
+        days-before-stale: 2
+        days-before-close: 5
+        days-before-ancient: 365
```
Comment (waahm7) [inline .github/workflows/stale_issue.yml:35]: Will this cause the issue to have a stale label after 2 days? We should probably remove this as well for now or increase the value.

Comment (jmklix) [reply .github/workflows/stale_issue.yml:35]: The `days-before-stale` only is triggered with the `response-requested` label. And the `days-before-close` is only triggered after the `closing-soon` label.

**Review by jbelkins — APPROVED** [review]

**Review by waahm7 — APPROVED** [review]

**Pull request merged**